src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,96 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.exeption.InvalidUserException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static RegistrationService registrationService;
+    private static User validUser;
+
+    @AfterEach
+    void clear() {
+        Storage.people.clear();
+    }
+
+    @BeforeAll
+    static void setUp() {
+        registrationService = new RegistrationServiceImpl();
+        validUser = new User("validlogin", "validpassword", 18);
+    }
+
+    @Test
+    void register_ValidUser_Ok() throws InvalidUserException {
```

Comment:
1 be more attentive to naming convention (small and capital letters).
fix the all similar places in the code below.
```suggestion
    void register_validUser_ok() {
```
2 remove throws `InvalidUserException`
fix the all places in the code.